cwf/gateway/integ_tests/Makefile

```diff
@@ -12,6 +12,7 @@ AUTH=Authenticate
 GX=Gx
 GY=Gy
 MULTISP=MultiSessionProxy
+OMNI=Omnipresent
```

Comment:
Don't think we need this because we only have a couple of tests for omni

Reply:
lets remove this and make OMNI part of all tests